TestRunner/TestRunner.cs

```diff
@@ -0,0 +1,76 @@
+using System.Reflection;
+using Attributes;
+
+namespace TestRunner;
+
+public class TestRunner
+{
+    public bool HasTested;
+    public readonly List<TestingClass> TestingClasses = new();
+
+    private void LoadTests(string path)
+    {
+        var types = Directory.EnumerateFiles(path, "*.dll", 0).Select(Assembly.LoadFrom)
+            .SelectMany(assembly => assembly.ExportedTypes);
+        if (!types.Any())
+        {
+            Console.WriteLine("Nothing to test");
+        }
+
+        foreach (var type in types)
+        {
+            if (!type.IsClass) continue;
+            var newTestingClass = new TestingClass(type);
+            var instance = Activator.CreateInstance(type);
+
+            TestingClasses.Add(newTestingClass);
+            newTestingClass.BeforeMethods = FindMethodsByAttrInClass(type, typeof(Before))
+                .ConvertAll(methodInfo => new NonTestMethod(methodInfo, instance, true));
+            newTestingClass.AfterMethods = FindMethodsByAttrInClass(type, typeof(After))
+                .ConvertAll(methodInfo => new NonTestMethod(methodInfo, instance, false));
+
+            var tests = FindMethodsByAttrInClass(type, typeof(MyTest));
+            foreach (var test in tests)
+            {
+                var attr = GetTestAttribute(test);
+                var a = (MyTest)attr;
+                newTestingClass.TestMethods.Add(new TestMethod(test, instance, a.expected, a.ignore != "", a.ignore));
```

Comment:
```suggestion
                newTestingClass.TestMethods.Add(new TestMethod(test, instance, a.expected, a.ignore != null, a.ignore));
```
Довольно странно считать, что ignore с пустым сообщением - это сигнал к запуску.
(-1)